scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -645,11 +640,6 @@ private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli d
 
             CrossgenUtil.CrossgenDirectory(c, outputDir);
 
-            // Generate .version file
-            var version = buildVersion.NuGetVersion;
-            var content = $@"{c.BuildContext["CommitHash"]}{Environment.NewLine}{version}{Environment.NewLine}";
-            File.WriteAllText(Path.Combine(outputDir, ".version"), content);
```

Comment:
No, the /dnvm directory will still contain version files; they are just populated all at once at the end now